src/lang/de.json

```diff
@@ -94,6 +94,7 @@
   "header_heroes":"Heroes",
   "header_blog":"Blog",
   "header_ingame":"Ingame",
+  "header_player_search": "Player Search",
```

Comment:
"header" is for top level navbar items.  "heading_" probably better choice here.